### PR TITLE
[MIG][9.0] product_sequence

### PR DESCRIPTION
**Pull request by SodexisTeam**

### CODE REVIEW

**Comment by atchuthan** [comment]
needs label change to "needs review"

**Comment by gurneyalex** [comment]
:+1:

File: product_sequence/README.rst [hunk 1]
```diff
@@ -13,6 +18,15 @@ they already have a unique reference (or no reference) set.  Products with a def
 
 Otherwise the setting of the unique constraint will fail and the module will fail to install.
 
+Usage
+=====
+
+#. Go to ...
```
Comment (pedrobaeza) [inline product_sequence/README.rst]: Please include short usage instructions

**Comment by pedrobaeza** [comment]
Please include some tests.

**Comment by atchuthan** [comment]
@pedrobaeza added test

Travis failure shows a strange error
product_sequence/tests/___init__.py:5:1: F401 '.test_product_sequence' imported but unused

File: product_sequence/__init__.py [hunk 2]
```diff
@@ -1,24 +1,7 @@
...
-#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
-#
-##############################################################################
+# -*- coding: utf-8 -*-
+# © 2004 Tiny SPRL
+# © 2016 Sodexis
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
 from . import models
 from .models.product_product import update_null_and_slash_codes
```
Comment (pedrobaeza) [inline product_sequence/__init__.py]: You should change this also to a post/pre_init_hook.

**Comment by pedrobaeza** [comment]
Please update .travis.yml definition to get a correct Travis status

**Comment by atchuthan** [comment]
@pedrobaeza corrections done

**Comment by pedrobaeza** [comment]
Why the coverage drop? You can test the init_hook creating a product, emptying by SQL the code, executing the hook and checking that is filled with the expected value.

**Comment by atchuthan** [comment]
@pedrobaeza added test and fixed your points.

COVERAGE DECREASED (-0.9%) TO 94.382%.. This seems alright to me :)

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**